app/web/css/pages/myConnections/myConnections.scss

```diff
@@ -7,3 +7,8 @@
 .connection__buttonWrapper {
   margin-top: 2rem;
 }
+
+.connection__deleteForm {
+  margin-left: 1.5rem;
+  width: calc((100% - 1.5rem) / 2);
```

Comment:
Does the line above work in all supported browsers, I saw it multiple times also in other files? Just wondering... Feel free to leave it as-is.

Reply:
This line does indeed, i have tested it in IE11 (which is the only one with partial support of the calc function), works like a charm.  Not so for the entire redesign, i'll have to make some commits to fix the errors on other parts of the page.